resources/views/templates/admin-user/edit.blade.php

```diff
@@ -0,0 +1,52 @@
+{{'@'}}extends('brackets/admin-ui::admin.layout.default')
+
+{{'@'}}section('title', trans('admin.{{ $modelLangFormat }}.actions.edit', ['name' => ${{ $modelVariableName }}->{{$modelTitle}}]))
+
+{{'@'}}section('body')
+
+    <div class="container-xl">
+
+        <div class="card">
+
+            @if($hasTranslatable)<{{ $modelJSName }}-form
+                :action="'{{'{{'}} ${{ $modelVariableName }}->resource_url }}'"
+                :data="{{'{{'}} ${{ $modelVariableName }}->toJsonAllLocales() }}"
+                :activation="!!'@{{ $activation }}'"
+                :locales="@{{ json_encode($locales) }}"
+                :send-empty-locales="false"
+                inline-template>
+            @else<{{ $modelJSName }}-form
+                :action="'{{'{{'}} ${{ $modelVariableName }}->resource_url }}'"
+                :data="{{'{{'}} ${{ $modelVariableName }}->toJson() }}"
+                :activation="!!'@{{ $activation }}'"
+                inline-template>
+            @endif
+
+                <form class="form-horizontal form-edit" method="post" {{'@'}}submit.prevent="onSubmit" :action="this.action">
+
+                    <div class="card-header">
+                        <i class="fa fa-pencil"></i> {{'{{'}} trans('admin.{{ $modelLangFormat }}.actions.edit', ['name' => ${{ $modelVariableName }}->{{$modelTitle}}]) }}
+                    </div>
+
+                    <div class="card-block">
```

Comment:
tu treba classu card-body namiesto card-block
```suggestion
                    <div class="card-body">
```